.github/workflows/dotnet.yml

```diff
@@ -8,22 +8,38 @@ on:
 
 jobs:
   build:
-
-    runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Setup .NET
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: 6.0.x
+        dotnet-version: |
+          6.0.x
+          7.0.x
     - name: Restore dependencies
-      run: dotnet restore src/FsExcel
+      run: dotnet restore
+      working-directory: src
```

Comment:
The core of the change is doing the dotnet commands on the sln file in src folder. Previously only lib code was built/restored